h2o-algos/src/main/java/hex/glm/GLMModel.java

```diff
@@ -1101,6 +1113,7 @@ public Submodel(double lambda , double alpha, double [] beta, int iteration, dou
   public final long    _nullDOF;
   public final double    _ySigma;
   public final long      _nobs;
+  public double[] _betaCndTemp;  // store temporary beta coefficients for checkpointing purposes
```

Comment:
confusing name, `temp` indicates something volatile, that is to be thrown away, what about `betaCoefsCheckpoint`?

Reply:
@honzasterba 

I changed the name to betaCndCheckpoint because it stored the last betaCnd.

Reply:
Changed name to _betaCndCheckpoint to store the latest state of _betaCnd